src/widgets/restFullClient/ui/BodyManager.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+import { Box, TextField } from '@mui/material';
+import TextFieldHint from '@shared/ui/TextFieldHint';
+import { IBodyProps } from '../model/bodyManagerModel';
+
+const BodyManager = (props: IBodyProps) => {
+  const { body, setBody } = props;
```

Comment:
```suggestion
const BodyManager = ({ body, setBody }: IBodyProps) => {
```

Или:
```suggestion
const BodyManager: FC<IBodyProps> = ({ body, setBody }) => {
```

Зависит от code style.
По хорошему JSDoc

```suggestion
/**
* Description
*/
const BodyManager = (props: IBodyProps) => {
  const { body, setBody } = props;
```

Reply:
исправил